bin/seed.js

```diff
@@ -0,0 +1,26 @@
+const mongoose = require("mongoose");
+const dbName = 'rol'
+const Boss = require("../models/user.model");
+mongoose.connect(`mongodb://localhost/${dbName}`, {
+    useNewUrlParser: true,
+    useUnifiedTopology: true,
+});
+const bcrypt = require("bcrypt")
+const bcryptSalt = 10
+const password = "diveloper"
+const salt = bcrypt.genSaltSync(bcryptSalt)
+const hashPass = bcrypt.hashSync(password, salt)
+const boss = [
+    {
+        username: "Admin",
+        name: "BOSS",
+        password: hashPass,
+        profileImg: undefined,
+        description: "El Gerente",
+        facebookId: undefined,
+        role: 'BOSS'
```

Comment:
Cuidado porque definiste en el modelo de otra forma los roles, no en mayúscula todo el rol.
```suggestion
        role: 'Boss'
```